SteamcLog/Classes/Helper/Config.swift

```diff
@@ -36,6 +36,9 @@ public struct Config {
     /// Toggles Sentry attaching stack traces  to errors. Default is true.
     internal let sentryAttachStacktrace: Bool
 
+    /// Toggles the ability to filter out errors from being reported to Sentry
+    internal let filterOut: FilterOut
```

Comment:
Do you think it's worth changing this to `[FilterOut]` to allow users to filter out multiple different types of errors?

Reply:
Users can already filter out multiple types of errors via the interface, is there a benefit to allowing multiple objects? It seems like the Filter interface could use nested functions for organization

Reply:
Oh of course, that works too 🚀